web/modules/custom/myroute/myroute.routing.yml

```diff
@@ -0,0 +1,17 @@
+myroute.practicingRoute:
+  path: '/routeit/practice'
+  defaults: 
+    _controller: 'Drupal\myroute\Controller\MyrouteController::practicingRoute'
+    _title: 'Route Content'
+  requirements:
+    _permission: 'access custom page'
+    _role: administrator + 	content_editor
+
+
+myroute.getArticleNo:
+  path: '/campaign/value/{id}'
```

Comment:
Please add proper validation on the id paramater.

Reply:
added